tests/suites/1_stateful/00_copy/00_0005_copy_from_stage_max_files.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+
+# Should be <root>/tests/data/
+
+for force in 'false'  'true'
+do
+	for purge in 'false'  'true'
+	do
+		table="test_max_files_force_${force}_purge_${purge}"
+		echo "drop table if exists ${table}" | $MYSQL_CLIENT_CONNECT
+		echo "CREATE TABLE ${table} (
+      id INT,
+      c1 INT
+    ) ENGINE=FUSE;" | $MYSQL_CLIENT_CONNECT
+	done
+done
+
+gen_files() {
+  rm -rf /tmp/00_0005
+  cp -r "$CURDIR"/../../../data/00_0005 /tmp
+}
+
+echo "drop stage if exists s5;" | $MYSQL_CLIENT_CONNECT
+echo "create stage s5 url = 'fs:///tmp/00_0005/data/' FILE_FORMAT = (type = PARQUET)" | $MYSQL_CLIENT_CONNECT
+
+
+for force in 'false'  'true'
+do
+	for purge in 'false'  'true'
+	do
+	  gen_files
+	  echo "select count(*) from @s5" | $MYSQL_CLIENT_CONNECT
```

Comment:
is " echo "select count(*) from @s5" | $MYSQL_CLIENT_CONNECT" on purpose of for debug?

Reply:
Yes. And I think we can save it.

Reply:
if you really want to add it,  better put it after `echo "--- force = ${force}, purge = ${purge}"`?

Reply:
which make results more readable.

Reply:
ok